src/app/components/login/login.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { LoginRoutingModule } from './login-routing.module';
+
+
+@NgModule({
+  declarations: [],
```

Comment:
You should declare the components that you use on this module in here rather than in the app module. That's the magic of lazy loading.